mongodbatlas/indexes.go

```diff
@@ -0,0 +1,93 @@
+package mongodbatlas
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+)
+
+const indexesPath = "groups/%s/clusters/%s/index"
+
+// IndexesService is an interface for interfacing with the clusters indexes
+// endpoints of the MongoDB Atlas API.
+// See more: https://docs.atlas.mongodb.com/reference/api/indexes/
+type IndexesService interface {
+	Create(context.Context, string, string, *IndexConfiguration) (*Response, error)
+}
+
+// IndexesServiceOp handles communication with the Cluster related methods
+// of the MongoDB Atlas API
+type IndexesServiceOp struct {
+	Client RequestDoer
+}
+
+var _ IndexesService = &IndexesServiceOp{}
+
+// IndexConfiguration represents current value of the metric that triggered the alert. Only present for alerts of type HOST_METRIC.
```

Comment:
Just a nit but this comment is confusing - this has nothing to do with metrics, yes?

Reply:
oops, probably copy/pasta, I'll fix this